playwright.config.js

```diff
@@ -95,7 +95,7 @@ const config = {
 
   /* Run your local dev server before starting the tests */
   webServer: {
-    command: 'cd test-apps/playwright && yarn start',
+    command: 'cd test-apps/playwright && yarn develop',
```

Comment:
should we test with yarn develop?

Reply:
So i thought no to begin with, but then realised we can't test the CTB without it 🙃

Reply:
true 😓